completions/fish/wl-paste.fish

```diff
@@ -0,0 +1,44 @@
+function __wayland_seats --description 'Print all wayland seats'
+    if type -q weston-info
+        weston-info 2>/dev/null | sed -n '/wl_seat/{n;s/\s*name: //;p}'
+    else if type -q loginctl
+        loginctl --no-legend --no-pager list-seats 2>/dev/null
+    else
+        # Fallback seat
+        echo "seat0"    
+    end
+end
+
+function __wl_paste_types --description 'Print types with context'
+    set -l type (commandline -o)
+
+    for i in (seq (count $type))
+        if [ "$type[$i]" = '--primary' ]; or [ "$type[$i]" = '-p' ]
+            set clip "primary"
+        else if [ "$type[$i]" = '--seat' ]; or [ "$type[$i]" = '-s' ]
+            set seat $type[(math $i + 1)]
+        end
+    end
+
+    # Note fish does not handle passing unset variables
+    # to commands well, thus setting clip to "--primary" and passing
+    # that to wl-paste wont work, so if statements are used instead
+    if test -n "$seat"; and test -n "$clip"
+        wl-paste --seat "$seat" -p -l
+    else if test -n "$seat"
+        wl-paste --seat "$seat" -l
+    else if test -n "$clip"
+        wl-paste -p -l
+    else
+        wl-paste -l
+    end
+end
+
+complete -c wl-paste -s h -l help -d 'Display a help message.'
```

Comment:
Should these descriptions end with a period? It looks like other commands don't do this.

Reply:
Fixed